lua/rainbow-delimiters/default.lua

```diff
@@ -27,7 +27,7 @@ local M = {
 		[''] = require 'rainbow-delimiters.strategy.global',
 	},
 	priority = {
-		[''] = 110,
+		[''] = (vim.hl or vim.highlight).priorities.treesitter + 100,
```

Comment:
Adding 100 to the `treesitter` priority would put us at 200 on Neovim 0.10.3, which is the `user` priority. How about instead we compute a value somewhere in-between `treesitter` and `semantic_tokens` based on whatever the current values are?

```suggestion
		local priorities = (vim.hl or vim.highlight).priorities
		[''] = vim.fn.round(priorities.semantic_tokens / 2 - priorities.treesitter / 2)
```

Reply:
Nice, that's smart

Reply:
Sorry, you are right. The formula is:

```
t + (s - t) / 2 = t + s/2 - t/2 = s/2 + t/2
```